core/src/test/scala/tech/beshu/ror/unit/acl/logging/AuditingToolTests.scala

```diff
@@ -16,143 +16,164 @@
-          ),
-          auditSink
-        )
-
-        auditingTool.audit(createAllowedResponseContext(Policy.Allow, Verbosity.Info)).runSyncUnsafe()
-      }
-      "request was matched by forbidden rule" in {
-        val auditSink = mock[AuditSinkService]
-        (auditSink.submit _).expects("test_2018-12-31", "mock-1", *).returning(())
-
-        val auditingTool = new AuditingTool(
-          AuditingTool.Settings(
-            RorAuditIndexTemplate.from("'test_'yyyy-MM-dd").right.get,
-            new DefaultAuditLogSerializer,
-            AuditCluster.LocalAuditCluster
-          ),
-          auditSink
-        )
-
-        val requestContext = MockRequestContext.indices.copy(timestamp = someday.toInstant, id = RequestContext.Id("mock-1"))
-        val responseContext = ForbiddenBy(
-          requestContext,
-          new Block(
-            Block.Name("mock-block"),
-            Block.Policy.Forbid,
-            Block.Verbosity.Info,
-            NonEmptyList.one(new MethodsRule(MethodsRule.Settings(NonEmptySet.one(Method.GET))))
-          ),
-          GeneralIndexRequestBlockContext(requestContext, UserMetadata.empty, Set.empty, List.empty, Set.empty, Set.empty),
-          Vector.empty
-        )
-
-        auditingTool.audit(responseContext).runSyncUnsafe()
-      }
-      "request was forbidden" in {
-        val auditSink = mock[AuditSinkService]
-        (auditSink.submit _).expects("test_2018-12-31", "mock-1", *).returning(())
-
-        val auditingTool = new AuditingTool(
-          AuditingTool.Settings(
-            RorAuditIndexTemplate.from("'test_'yyyy-MM-dd").right.get,
-            new DefaultAuditLogSerializer,
-            AuditCluster.LocalAuditCluster
-          ),
-          auditSink
-        )
-
-        val requestContext = MockRequestContext.indices.copy(timestamp = someday.toInstant, id = RequestContext.Id("mock-1"))
-        val responseContext = Forbidden(requestContext, Vector.empty)
-
-        auditingTool.audit(responseContext).runSyncUnsafe()
-      }
-      "request was finished with error" in {
-        val auditSink = mock[AuditSinkService]
-        (auditSink.submit _).expects("test_2018-12-31", "mock-1", *).returning(())
-
-        val auditingTool = new AuditingTool(
-          AuditingTool.Settings(
-            RorAuditIndexTemplate.from("'test_'yyyy-MM-dd").right.get,
-            new DefaultAuditLogSerializer,
-            AuditCluster.LocalAuditCluster
-          ),
-          auditSink
-        )
-
-        val requestContext = MockRequestContext.indices.copy(timestamp = someday.toInstant, id = RequestContext.Id("mock-1"))
-        val responseContext = Errored(requestContext, new Exception("error"))
-
-        auditingTool.audit(responseContext).runSyncUnsafe()
+    "log sink is used" should {
+      "saved audit log to file defined in log4j config" in {
+        for {
+          file <- File("./core/build/tmp/readonlyrest_audit.log").toTemporary
```

Comment:
we have a hardcoded core module name here. Maybe we should take advantage of better.files' temporary files?